tideways.c

```diff
@@ -3376,7 +3376,7 @@ PHP_FUNCTION(tideways_span_callback)
 		Z_TRY_ADDREF(fci.function_name);
 
 		if (fci.object != NULL) {
-			fci.object->gc.refcount++;
+			GC_REFCOUNT(fci.object)++;
```

Comment:
The internal details of this may change (however unlikely), this is just more forward compatible ...